pageserver/src/http/openapi_spec.yml

```diff
@@ -354,6 +354,52 @@ paths:
               schema:
                 $ref: "#/components/schemas/Error"
 
+  /v1/tenant/{tenant_id}/size:
+    parameters:
+      - name: tenant_id
+        in: path
+        required: true
+        schema:
+          type: string
+          format: hex
+    get:
+      description: |
+        Calculate tenant's size.
```

Comment:
Let's also mention what units the size has.

Reply:
In the next push, elaborated on the description and the actual field.